### PR TITLE
Keeps the estimates real.

### PR DESCRIPTION
**Pull request by skellock**
`npm` is slow.   `¯\_(ツ)_/¯`

### CODE REVIEW

File: ignite-generator/src/app/index.es [hunk 1]
```diff
@@ -367,7 +367,7 @@ export class AppGenerator extends Generators.Base {
    * Yeoman times its template copies.  :(
    */
   install () {
-    const npmStatus = 'Installing Ignite dependencies (~30 seconds-ish)'
+    const npmStatus = 'Installing Ignite dependencies (~ 1 minute)'
```
Comment (GantMan) [inline ignite-generator/src/app/index.es]: I want the `-ish` PUT IT BACK!

I ❤️  it

Comment (skellock) [inline ignite-generator/src/app/index.es]: haha. that's the ~ means.  =)

Comment (GantMan) [inline ignite-generator/src/app/index.es]: I don't care, had no idea I loved this until I saw ish, it makes me smile.   I'm sorry but I require it to stay in my life.  Don't make ish and I run off to vegas.

Comment (skellock) [inline ignite-generator/src/app/index.es]: I care that you care.  I will bring back the ish.

**Comment by GantMan** [comment]
# Together Again!
image

**Comment by skellock** [comment]
HAHAHHAHAHAHHAHAHAHAHHAHAHAHAHAHHAHAHA.

Comment of the year.

**Pull request merged**